samples/sample-custom-subscription-hook/package.json

```diff
@@ -8,13 +8,11 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/node": "^20.3.1",
     "@types/react": "^18.2.13",
-    "@types/react-dom": "^18.2.6",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "bigbluebutton-html-plugin-sdk": "0.0.73",
     "path": "^0.12.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-modal": "^3.16.1",
```

Comment:
You removed `react-modal` as a dependency but didn't removed its usage in https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/blob/f1f32b1fb52a943805e9f842d1a393ee04c39e01/samples/sample-custom-subscription-hook/src/components/sample-custom-presentation-subscription-plugin-item/component.tsx#L3.